manticore/platforms/linux.py

```diff
@@ -292,6 +292,9 @@ class Linux(Platform):
     This class emulates the most common Linux system calls
     '''
 
+    # from /usr/include/asm-generic/resource.h
+    RLIMIT_NOFILE = 7 #/* max number of open files */
```

Comment:
just a question: 7 is the max number of file a process can have open? doesn't this seem really low?

Reply:
No, 7 is the magic value that refers to `RLIMIT_NOFILE ` constant; i didn't include others from the header since they're not used yet.